index.html

```diff
@@ -17,21 +17,27 @@
             </nav>
         </header>
         <main>
-        <h1>Welcome to Alain CSD120 Website!</h1>
-            <p>I'm originally a Java programmer who really enjoyed working with AI and game logic. Now I dabble in HTML, and I am learning python. I have some experience from high school coding an AI, I worked on a project alone where I would construct an AI to play Starcraft. I was heading toward engineering in highschool, but coding took the cake.</p>
-            <p>In my free time you will find me either walking my dog, or playing video games. I like to work alone and enjoy problem solving complex issues.</p>
-        <h2>My three favorite games</h2>
-        <ol>
-            <li>Overwatch</li>
-            <li>Rainbow Six : Siege</li>
-            <li>Escape From Tarkov</li>
-        </ol>
-        <h2>Haiku</h2>
-        <p><span style = "background-color:#7887AB">Objections to lies
-            <br>Conjectures to find the truth
-            <br>Find the right verdict!</span>
-            <br><i><span class="byline">by <b>Suzie </span></b></i>
```

Comment:
Invalid syntax (nested elements must be closed before their parent elements are closed)